tokio/src/net/tcp/stream.rs

```diff
@@ -155,7 +155,7 @@ impl TcpStream {
 
     pub(crate) fn new(connected: mio::net::TcpStream) -> io::Result<TcpStream> {
         let io = PollEvented::new(connected)?;
-        Ok(new_instrumented_resource!(Network, TcpStream { io }))
+        Ok(new_instrumented_resource!("network", TcpStream { io }))
```

Comment:
nit/tioli: maybe
```suggestion
        Ok(new_instrumented_resource!("net", TcpStream { io }))
```
for consistency with the module naming scheme?